environment.yml

```diff
@@ -0,0 +1,51 @@
+name: holoplot
```

Comment:
It might be good to clarify what this environment is for; is it just for running through the examples or for building the docs?

Reply:
Building docs and testing.